app/helpers/calagator/mapping_helper.rb

```diff
@@ -22,11 +22,8 @@ def map_provider_dependencies
         "https://maps.googleapis.com/maps/api/js?key=#{SECRETS.mapping_google_maps_api_key}&sensor=false",
         "leaflet_google_layer",
       ],
-    }[map_provider]
-  end
-
-  def mapping_js_includes
-    leaflet_js + map_provider_dependencies
```

Comment:
The leaflet javascript library is used for all of the map providers, so we can't combine these two methods. Instead, `#map_provider_dependencies` should be updated to return an empty array instead of nil, so the `+` in `#mapping_js_includes` works consistently.